RedHat-UX/red-hat-design-system
# docs: split Color page best practices into dos and dont's

## What I did

1. Updated the best practices section on the Usage subpage


## Testing Instructions

1. Check [deploy preview](https://deploy-preview-1341--red-hat-design-system.netlify.app/foundations/color/usage/).

## Notes to Reviewers
